installing/installing_with_agent_based_installer/prepare-pxe-assets-agent.adoc

```diff
@@ -34,7 +34,11 @@ include::modules/pxe-assets-ocp-agent.adoc[leveloffset=+1]
 
 // Manually adding IBM Z agents
 include::modules/installing-ocp-agent-ibm-z.adoc[leveloffset=+1]
+
+// Adding {ibm-z-title} agents with z/VM
 include::modules/installing-ocp-agent-ibm-z-zvm.adoc[leveloffset=+2]
+
+// Adding {ibm-z-name} agents with {op-system-base} KVM
```

Comment:
I do not want to hold up your PR, but might be worthwhile checking with Silke if RHEL KVM remains valid for 4.15. I removed its reference from the Welcome page based on feedback in this [G. Doc](https://docs.google.com/document/d/1UTtAuYfRjOEyVw-rh7J7GCmY8WPgH_9ICTW1l3HWPsU/edit) (ssearch for RHEL KVM).

Reply:
cc @SNiemann15. 

I think since this content already exists in the live doc I may as well continue with my improvements to it, but that's a good thing to check in general.

Reply:
@skopacz1 @dfitzmau Yes installation on RHEL KVM is still valid for IBM Z. The reason for removing the entry on the welcome page is that I retructured the IBM Z installation docs to have on single entry point.